service_container/3.3-di-changes.rst

```diff
@@ -577,6 +577,13 @@ Start by updating the service ids to class names:
     account when looking up for services.
     Furthermore it is always recommended to check for definition existence
     using ``has()`` function.
+    
+.. note::
+
+    If you get rid of deprecations and extend `AbstractController` instead of `Controller` for
```

Comment:
In the RST format used in Symfony Docs, code uses two backticks instead of one (like in Markdown):

```rst
... ``AbstractController`` instead of ``Controller`` ...
```